c_src/enacl_nif.c

```diff
@@ -886,6 +955,9 @@ static ErlNifFunc nif_funcs[] = {
 	{"crypto_sign_open_b", 2, enif_crypto_sign_open},
 	{"crypto_sign_open", 2, enif_crypto_sign_open, ERL_NIF_DIRTY_JOB_CPU_BOUND},
 
+        {"crypto_sign_detached", 2, enif_crypto_sign_detached, ERL_NIF_DIRTY_JOB_CPU_BOUND},
```

Comment:
If you want to create the interpreter bound variants later, look at the code path for `crypto_sign_open_b`.